usr/src/tools/ctf/Makefile

```diff
@@ -45,9 +44,28 @@ ctfdiff: libctf
 ctfdump: libctf
 ctfconvert: libctf
 
-all clean clobber install lint: dwarf .WAIT $(SUBDIRS)
+all clean clobber install: dwarf .WAIT $(SUBDIRS)
 
 dwarf $(SUBDIRS): FRC
 	@cd $@; pwd; $(MAKE) $(TARGET)
 
+#
+# This is a somewhat arbitrary list.
+#
+CTFTOOLS_FILES = \
+	bin/i386/ctfconvert \
+	bin/i386/ctfdiff \
+	bin/i386/ctfdump \
+	bin/i386/ctfstabs \
+	bin/i386/ctfstrip \
+	lib/i386/libctf.so \
+	lib/i386/libctf.so.1 \
+	lib/i386/libdwarf.so \
+	lib/i386/libdwarf.so.1 \
+	lib/i386/libelf.so \
+	lib/i386/libelf.so.1
+
+ctftools: install
+	cd $(ROOTONBLD) && $(TAR) cvzf $(CTFTOOLS_TARBALL) $(CTFTOOLS_FILES)
```

Comment:
Where are we defining CTFTOOLS_TARBALL?

Reply:
It's in the environment when we build this target.

Reply:
Ah got it - smartos-live.git:tools/build_ctftools  Could we add a comment here saying where that came from?